src/main/java/com/pubnub/api/managers/DelayedReconnectionManager.java

```diff
@@ -61,6 +64,10 @@ private boolean isReconnectionPolicyUndefined() {
         return false;
     }
 
+    private double getRandomDelayInMilliSeconds() {
```

Comment:
change return type to int (since you're calling nextInt anyway)

Reply:
Good catch.